{"name": "test_index_page", "status": "broken", "statusDetails": {"message": "KeyError: 'DB_HOST'", "trace": "client = <FlaskClient <Flask 'app'>>\n\n    def test_index_page(client):\n        \"\"\"Test that index page loads correctly\"\"\"\n>       rv = client.get('/')\n\ntests/test_app.py:13: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../../.local/lib/python3.10/site-packages/werkzeug/test.py:1127: in get\n    return self.open(*args, **kw)\n../../../.local/lib/python3.10/site-packages/flask/testing.py:216: in open\n    return super().open(  # type: ignore\n../../../.local/lib/python3.10/site-packages/werkzeug/test.py:1072: in open\n    response = self.run_wsgi_app(request.environ, buffered=buffered)\n../../../.local/lib/python3.10/site-packages/werkzeug/test.py:943: in run_wsgi_app\n    rv = run_wsgi_app(self.application, environ, buffered=buffered)\n../../../.local/lib/python3.10/site-packages/werkzeug/test.py:1229: in run_wsgi_app\n    app_rv = app(environ, start_response)\n../../../.local/lib/python3.10/site-packages/flask/app.py:2088: in __call__\n    return self.wsgi_app(environ, start_response)\n../../../.local/lib/python3.10/site-packages/flask/app.py:2073: in wsgi_app\n    response = self.handle_exception(e)\n../../../.local/lib/python3.10/site-packages/flask/app.py:2070: in wsgi_app\n    response = self.full_dispatch_request()\n../../../.local/lib/python3.10/site-packages/flask/app.py:1515: in full_dispatch_request\n    rv = self.handle_user_exception(e)\n../../../.local/lib/python3.10/site-packages/flask/app.py:1513: in full_dispatch_request\n    rv = self.dispatch_request()\n../../../.local/lib/python3.10/site-packages/flask/app.py:1499: in dispatch_request\n    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)\napp.py:18: in index\n    conn = get_db_connection()\napp.py:9: in get_db_connection\n    host=os.environ['DB_HOST'],\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = environ({'SHELL': '/bin/bash', 'SESSION_MANAGER': 'local/alekspopov:@/tmp/.ICE-unix/2006,unix/alekspopov:/tmp/.ICE-uni...DATAFILE': '/home/aleks/mipt/pdris/web/.coverage', 'PYTEST_CURRENT_TEST': 'tests/test_app.py::test_index_page (call)'}), key = 'DB_HOST'\n\n    def __getitem__(self, key):\n        try:\n            value = self._data[self.encodekey(key)]\n        except KeyError:\n            # raise KeyError with the original key value\n>           raise KeyError(key) from None\nE           KeyError: 'DB_HOST'\n\n/usr/lib/python3.10/os.py:680: KeyError"}, "description": "Test that index page loads correctly", "start": 1734217317466, "stop": 1734217317467, "uuid": "69cfe5f1-8ffc-469d-b174-3532182b19c3", "historyId": "eb1ca4e47acbe43ac52029c0fe57c217", "testCaseId": "eb1ca4e47acbe43ac52029c0fe57c217", "fullName": "tests.test_app#test_index_page", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_app"}, {"name": "host", "value": "alekspopov"}, {"name": "thread", "value": "30606-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_app"}]}